wave_lang/kernel/wave/wave.py

```diff
@@ -813,6 +814,16 @@ def _trace_and_get_kernel_signature(
             partial(generate_bounds_exprs, trace, self.constraints),
         ]
 
+        graph_passes.append(
+            partial(
+                location_check_pass,
+                trace,
+                "enforce-locations",
+                log=False,
+                enforce_100=options.enforce_locations,
```

Comment:
Question: what is enforce_100?

Reply:
Enforce 100% locations.  I can rename that -- the name makes sense in the context of the pass, but for external users, probably `enforce_locations` is better.